internal/otgutils/print.go

```diff
@@ -0,0 +1,70 @@
+package otgutils
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/ondatra"
+	otgtelemetry "github.com/openconfig/ondatra/telemetry/otg"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// LogFlowMetrics is displaying the otg flow statistics.
+func LogFlowMetrics(t testing.TB, otg *ondatra.OTG, c gosnappi.Config) {
```

Comment:
in latest Ondatra and feature profiles, references to ondatra.OTG are now otg.OTG